src/Stormpath/Resource/CustomData.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace Stormpath\Resource;
+
+
+use Stormpath\Stormpath;
+
+class CustomData extends AbstractCollectionResource implements Saveable {
+    const PATH = 'customData';
+
+    public function __get($key)
+    {
+       return $this->getProperty($key);
+    }
+
+    public function __set($key, $value)
+    {
+        $this->setProperty($key, $value);
+    }
+
+    public function getItemClassName()
+    {
+        return Stormpath::CUSTOM_DATA;
+    }
+
+    public function save()
+    {
+        return $this->getDataStore()->save($this);
+    }
+
+    public function delete()
```

Comment:
I think this method won't be necessary when changing the parent to be the instance resource class.